fyle/platform/apis/v1beta/admin/tax_groups.py

```diff
@@ -2,6 +2,8 @@
 V1 Beta Admin Tax Groups
 """
 
+from typing import Dict, List
+from fyle.platform.internals.post_bulk_resources import PostBulkResources
```

Comment:
There is no use of `PostBulkResources` and the way it is, is not reusable, meaning we'll have to do this for every class for bulk post. Can you think of a common approach that can be done here.